pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.black]
+line-length = 119
+target-version = ['py37']
```

Comment:
I believe this should be `py38` as we recently dropped 3.7 support https://github.com/lvwerra/trl/pull/441

Reply:
Done! Thanks for catching that